{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/users/auth/api.py

```diff
@@ -36,3 +39,49 @@ def register(self, request):
         user = create_user_account(**serializer.validated_data)
         data = serializers.AuthUserSerializer(user).data
         return response.Created(data)
+
+    @list_route(['POST', ])
+    def logout(self, request):
+        """
+        Calls Django logout method and delete the Token object
```

Comment:
This doesn't delete the token, so the docs need to accurate as per the implementation.